CodeToReview.java

```diff
@@ -0,0 +1,250 @@
+        while (true) {
+            switch (inO) {
+                case 1:
+                    if (a1 == "1") {
+                        a1 = "O";
+                    }
+                    break;
+                case 2:
+                    if (a2 == "2") {
+                        a2 = "O";
+                    }
+                    break;
+                case 3:
+                    if (a3 == "3") {
+                        a3 = "O";
+                    }
+                    break;
+                case 4:
+                    if (a4 == "4") {
+                        a4 = "O";
+                    }
+                    break;
+                case 5:
+                    if (a5 == "5") {
+                        a5 = "O";
+                    }
+                    break;
+                case 6:
+                    if (a6 == "6") {
+                        a6 = "O";
+                    }
+                    break;
+                case 7:
+                    if (a7 == "7") {
+                        a7 = "O";
+                    }
+                    break;
+                case 8:
+                    if (a8 == "8") {
+                        a8 = "O";
+                    }
+                    break;
+                case 9:
+                    if (a9 == "9") {
+                        a9 = "O";
+                    }
+                    break;
+                default:
+                    System.out.printf("An error occured. Please accept it politely and restart the game.");
+                    break;
+            }
+            System.out.printf("\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n");
+            if (null != Winners(a1,a2,a3,a4,a5,a6,a7,a8,a9))switch (Winners(a1,a2,a3,a4,a5,a6,a7,a8,a9)) {
+                case "NONE":
+                    if (a1 != "1" && a2 != "2" && a3 != "3" && a4 != "4" && a5 != "5" && a6 != "6" && a7 != "7" && a8 != "8" && a9 != "9") {
+                        winner = true;
+                        System.out.println("DRAW!");
+                        Thread.sleep(3000);
+                        break;
+                    } else {
+                        PrintBoard(a1,a2,a3,a4,a5,a6,a7,a8,a9);
+                        System.out.printf("Player X's turn: ");
+                        inX = input.nextInt();
+                        break;
+                    }
+                case "X":
+                    winner = true;
+                    System.out.println("X HAS WON!!!");
+                    Thread.sleep(3000);
+                    break;
+                case "O":
+                    winner = true;
+                    System.out.printf("O HAS WON!!!");
+                    Thread.sleep(3000);
+                    break;
+                default:
+                    break;
+            }
+            if (winner) {
+                break;
+            }
+            switch (inX) {
```

Comment:
Even in this switch case, in all cases variable is assigned X value. Switch case doesn't make any sense if all the cases are doing the same thing. If-else is the better option